pkg/identityserver/gateway_registry_test.go

```diff
@@ -207,6 +207,56 @@ func TestGatewaysCRUD(t *testing.T) {
 			a.So(updated.Name, should.Equal, "Updated Name")
 		}
 
+		t.Run("contact_info_restrictions", func(t *testing.T) { // nolint:paralleltest
```

Comment:
Spaces are automatically converted to underlines, so you can just leave the spaces in the naming, or use something like `ContactInfoRestrictions`. We don't have a consistent style in the codebase.

Reply:
Was using the underscore to make it easier to skip the whole test name under my text editor, since I don't want to create a third way of doing things I changed to the first suggestion. Thanks for catching that.